submodules/islandora_webform_ingest/includes/utilities.inc

```diff
@@ -422,23 +422,32 @@ function iwi_webform_ingest_settings($webform_nid) {
  *   either "append" or "replace" (default)
  * @param string $delimiter
  *   the character used to delimit the $elementspath
+ * @param int $offset
+ *   when == 0 will act as a replace, higher values will append
+ *   if the element does not exist. If already in place will replace.
+ *   Must be in range of [0,10], being 1 the default.
  */
-function iwi_set_form_element_byhash(&$form, &$form_state, &$cloned_elements, $elementspath, $input_value, $mode = 'replace', $delimiter = ':') {
+function iwi_set_form_element_byhash(&$form, &$form_state, $elementspath, $input_value, $mode = 'replace', $delimiter = ':', $offset = 1) {
```

Comment:
Should we move $delimiter to the end, so that it doesn't always need to be provided when we are using an $offset?